useful_resources.rst

```diff
@@ -27,3 +27,52 @@ An interesting `Behat and Mink cheat sheet`_ developed by `Jean-François Lépin
 .. _`blog post`: http://blog.jetbrains.com/phpstorm/2014/07/using-behat-in-phpstorm/
 .. _`Behat and Mink cheat sheet`: http://blog.lepine.pro/images/2012-04-behat-cheat-sheet1.pdf
 .. _`Jean-François Lépine`: http://blog.lepine.pro
```

Comment:
those link references should stay at the bottom of the file. the new section should be added before them.